lib/block-supports/layout.php

```diff
@@ -35,10 +35,11 @@ function gutenberg_register_layout_support( $block_type ) {
  * @param boolean $should_skip_gap_serialization Whether to skip applying the user-defined value set in the editor.
  * @param string  $fallback_gap_value            The block gap value to apply.
  * @param array   $block_spacing                 Custom spacing set on the block.
+ * @param boolean $should_enqueue              Whether to enqueue the styles for rendering.
```

Comment:
nitpick: alignments 😅 
```suggestion
 * @param boolean $should_enqueue                Whether to enqueue the styles for rendering.
```